### ISSUE TITLE
Integrate Redis to cache queries

### ISSUE THREAD

**Issue by byliuyang** [byliuyang/short#198]

**Comment by lawaloy** [comment]
Can i take on this?

**Comment by byliuyang** [comment]
@lawaloy Have you done system design before?

**Comment by lawaloy** [comment]
Studying grokking the system design 

Kind Regards,

Oluwaseyi Lawal  
 
  On Sun, Oct 13, 2019 at 6:20 PM, Harry Liu<notifications@github.com> wrote:   
@lawaloy Have you done system design before?

**Comment by byliuyang** [comment]
@lawaloy Please propose and discuss your design here. Please also link any library you are planing to use. Docker compose file would also be changed. I personally think this is a difficult architecture & design issue.

**Comment by lawaloy** [comment]
I'm trying to use this link to get started: https://www.npmjs.com/package/apicache 
 
  On Sun, Oct 13, 2019 at 6:45 PM, Harry Liu<notifications@github.com> wrote:   
@lawaloy Please propose and discuss your design here. Please also link any library you are planing to use. Docker compose file would also be changed. I personally think this is a difficult architecture & design issue.

**Comment by byliuyang** [comment]
@lawaloy Well, this library looks cool 😄 However, the library is for Node.js. Our backend is written in Golang.

**Comment by lawaloy** [comment]
Yes, I know this node.js and short's backend is goland. I said in my last email that I am using this approach for tracking.

Kind Regards,

Oluwaseyi Lawal  
 
  On Mon, Oct 14, 2019 at 10:52 AM, Harry Liu<notifications@github.com> wrote:   
@lawaloy Well, this library looks cool 😄 However, the library is for Node.js. Our backend is written in Golang.
